### DISCUSSION TITLE
tailwindcss v3.0.5 is not picking up changes. Just the styles I had used before. Not other frameworks installed

### DISCUSSION THREAD

**Discussion by jamescripto** [tailwindlabs/tailwindcss#12293] [Help]
I have only Tailwind for this project. Not other frameworks. Some default colours stop working and then I realised just previously used colors are working. New default colours are not updated  on the main.css

**Comment by alex-krasikau** [comment]
Hey @jamescripto!

Please follow this guide and ensure that the setup is correct - https://dev.to/plainsailing/tailwind-css-not-working-its-probably-this-9m4? I'm almost certain that the reason is that you are using the classes in a file that is located outside the scope of the `content` configuration.
